tools/docker-compose/ansible/roles/sources/templates/nginx.locations.conf.j2

```diff
@@ -10,7 +10,7 @@ location {{ (ingress_path + '/favicon.ico').replace('//', '/') }} {
     alias /awx_devel/awx/public/static/favicon.ico;
 }
 
-location ~ ^({{ (ingress_path + '/websocket/').replace('//', '/') }}|{{ (ingress_path + '/api/websocket/').replace('//', '/') }}) {
+location ~ ^({{ (ingress_path + '/websocket/').replace('//', '/') }}|{{ (ingress_path + '/api/websocket/').replace('//', '/') }}|{{ (ingress_path + '/api/controller/v2/websocket/').replace('//', '/') }}) {
```

Comment:
We don't want to hard code in `controller` into the awx project. 
We will need to create a new playbook variable and use it here. That same variable should also set `settings.OPTIONAL_API_URLPATTERN_PREFIX = {{ some_playbook_var }}`

Reply:
@chrismeyersfsu where exactly? I am not familiar with your docker-compose tools and I'm unsure where its best appropriate to add this variable.